src/test/java/net/snowflake/client/jdbc/ResultSetAsyncIT.java

```diff
@@ -216,7 +216,9 @@ public void testGetMethods() throws Throwable {
     clob.setString(1, "hello world");
     Statement statement = connection.createStatement();
     statement.execute(
-        "create or replace table test_get(colA integer, colB number, colC number, colD string, colE double, colF float, colG boolean, colH text, colI binary(3), colJ number(38,9), colK int, colL date, colM time, colN timestamp_ltz)");
+        "create or replace table test_get(colA integer, colB number, colC number, colD string,"
```

Comment:
Same, let's get rid of these style changes. I am fine with making these changes but let's not do all of thi sin the same PR